Source/WebCore/css/calc/CSSCalcOperationNode.cpp

```diff
@@ -650,26 +650,19 @@ void CSSCalcOperationNode::combineChildren()
     m_isRoot = IsRoot::No;
     
     if (m_children.size() < 2) {
-        if (m_children.size() == 1 && isTrigNode()) {
+        if (isTrigNode() || isExpNode() || calcOperator() == CalcOperator::Sqrt) {
             double resolvedValue = doubleValue(m_children[0]->primitiveType());
             auto newChild = CSSCalcPrimitiveValueNode::create(CSSPrimitiveValue::create(resolvedValue));
             m_children.clear();
             m_children.append(WTFMove(newChild));
         }
-        
-        if (isExpNode()) {
```

Comment:
I think we can do a similar refactoring for stepped node and round node below.

Reply:
Done.